packages/plugin-core/src/_extension.ts

```diff
@@ -157,6 +157,16 @@ async function startServerProcess(): Promise<{
   if (port) {
     return { port };
   }
+
+  // if in dev mode, simplify debugging without going multi process
+  if (getStage() !== "prod") {
+    const out = await launchv2({
+      logPath: path.join(__dirname, "..", "..", "dendron.server.log"),
+    });
+    return {port: out.port}
+  }
+
+  // start server is separate process
   const logPath = DendronWorkspace.instance().context.logPath;
   const out = await ServerUtils.execServerNode({
```

Comment:
do we need to similarly put this inside an `else{` block; i.e. only launch the background process if we're prod?

Reply:
nope. we should never hit this in development

Reply:
Ah, misread the code logic.